src/initFirebase.ts

```diff
@@ -0,0 +1,12 @@
+import firebase from 'firebase';
+
+const config = {
+    apiKey: process.env.REACT_APP_FIREBASE_API_KEY,
```

Comment:
`process.env.REACT_APP_FIREBASE_API_KEY` where do you store it?

Reply:
in the .env.local file in my root directory for creating environment variables of the app. Is it good solution or better store directly in the config variable?

Reply:
1. It is good solution, but please take into consideration, that I or any other person might would like to run the app locally on personal laptop. The app won't work fully.
2. I don't see any .env file in the repository. What else this file can contain?

Reply:
.env file was in the gitignore file(that why git ignored it), overwise I added firebaseConfig file in separate folder, so for now everyone can run the app locally